app/assets/stylesheets/components/_btn.scss

```diff
@@ -23,3 +23,8 @@
     background-color: transparent;
   }
 }
+
+.usa-button:disabled.usa-button--active,
+.usa-button--disabled.usa-button--active {
+  @include set-text-and-bg('primary-darker', $context: 'Button');
```

Comment:
👀 I have never seen kwards in SCSS like this, what a time to be alive

Reply:
👍 For context, the general approach here is just to repeat the same styles as the [default active styles from USWDS](https://github.com/uswds/uswds/blob/846e45b50662851a6d37f20726d9d20dd415a3fa/packages/usa-button/src/styles/_usa-button.scss#L42), but with higher specificity to make sure the active styles override the disabled styles.